Riot/Modules/Room/VoiceMessages/VoiceMessageMediaServiceProvider.swift

```diff
@@ -123,6 +125,10 @@ import MediaPlayer
         pauseAllServicesExcept(nil)
     }
     
+    func setNowPlayingInfoProvider(_ provider: VoiceMessageNowPlayingInfoProvider, forPlayer player: VoiceMessageAudioPlayer) {
```

Comment:
I would rename this to `registerNowPlayingInfoProvider` as set implies there's only one of them. Preusumably there shoud be a `deregister` counterpart too called at appropriate times, but I guess we could rely on the weakened `mapTable` dropping them eventually.